src/index.html

```diff
@@ -10,7 +11,50 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      action="#"
+      method="post"
+      data-qa="big"
+      name="bigForm"
+    >
+      <label class="searchBar">
+        <img
+          src="images/Search.svg"
+          alt="Search image"
+          class="searchBar__img--big"
```

Comment:
use common BEM element together with modifier an put common styles to searchBar__img 
```suggestion
          class="searchBar__img searchBar__img--big"
```